### PR TITLE
Reducer testing tj

### LINKED ISSUE #39: Reducers testing
As a dev I want to make sure that my reducers are tested

### PR DESCRIPTION
**Pull request by taylorjohnson141**
### What does this PR do?
Adds testing for all reducers

### Where should the reviewer start?
Reducers.js

### How should this be manually tested?
npm test

### Any background context you want to provide?
All should pass

### What are the relevant tickets?
closes #39 

### Questions:

Are their any other test the reducers need?

### CODE REVIEW

**Review by timkeresey — COMMENTED** [review]
This would be the extent of reducer testing as far as I can tell, but we can take another look tomorrow to make sure.

File: src/reducers/addIDReducer.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  it('should return an empty array if action.type !== ADD_ID',()=>{
+    expect(addID(undefined,action)).toHaveLength(0)
+  })
+  it('should return a array with a value if action.type === ADD_ID',()=>{
+    expect(addID(undefined,rightAction)).toStrictEqual([{id:1,name:'John'}])
+  })
+  it('should be able to add to state if state is defined',()=>{
+    expect(addID([{id:2,name:'Kurt'}],rightAction)).toStrictEqual([{id:2,name:'Kurt'},{id:1,name:'John'}])
+  })
+  it('should be able to add multiple identites',()=>{
```
Comment (ericberglund117) [inline src/reducers/addIDReducer.test.js:21]: This looks great! Awesome job testing different edge cases

File: src/reducers/isLoadingReducer.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  let goodAction2 = {
+    type:'IS_LOADING',
+    isLoading:false
+  }
+  it('should return false if input action.type !== IS_LOADING',() =>{
+    expect(isLoading(undefined,dummyAction)).toBe(false)
+  })
+  it('should return action.state if input action.type === IS_LOADING',() =>{
+    expect(isLoading(undefined,goodAction1)).toBe(true)
+    expect(isLoading(undefined,goodAction2)).toBe(false)
```
Comment (ericberglund117) [inline src/reducers/isLoadingReducer.test.js:21]: You might have to walk me through the undefined parameter, but this still looks good!

**Review by ericberglund117 — APPROVED** [review]
